docker/scripts/checkCustomFiles.sh

```diff
@@ -0,0 +1,6 @@
+if [[ -e /custom_files/ ]]; then
+	source /root/.bash_profile; \
+	sleep 5; \
+	/bin/cp --force /custom_files/cbiologo.png $CATALINA_HOME/webapps/cbioportal/images;
```

Comment:
100% agree to modify using the cBioSkins. Only thing required is to pass the customization filepath using the -v during runtime and updating the portal.properties file accordingly